src/files/XmlFile.ts

```diff
@@ -623,51 +447,15 @@ export class XmlFile {
      * Convert the brightscript/brighterscript source code into valid brightscript
      */
     public transpile(): CodeWithSourceMap {
-        this.logDebug('transpile');
-        //eventually we want to support sourcemaps and a full xml parser. However, for now just do some string transformations
-        let chunks = [] as Array<SourceNode | string>;
-        for (let i = 0; i < this.lines.length; i++) {
-            let line = this.lines[i];
-            let lowerLine = line.toLowerCase();
-
-            let componentLocationIndex = lowerLine.indexOf('</component>');
-            //include any bs import statements
-            if (componentLocationIndex > -1) {
-                let missingImports = this.getMissingImportsForTranspile()
-                    //change the file extension to .brs since they will be transpiled
-                    .map(x => x.replace('.bs', '.brs'));
-                //always include the bslib file
-                missingImports.push('source/bslib.brs');
-
-                for (let missingImport of missingImports) {
-                    let scriptTag = `<script type="text/brightscript" uri="${util.getRokuPkgPath(missingImport)}" />`;
-                    //indent the script tag
-                    let indent = ''.padStart(componentLocationIndex + 4, ' ');
-                    chunks.push(
-                        '\n',
-                        new SourceNode(1, 0, this.pathAbsolute, indent + scriptTag)
-                    );
-                }
-            } else {
-                //we couldn't find the closing component tag....so maybe there's something wrong? or this isn't actually a component?
-            }
-
-            //convert .bs extensions to .brs
-            let idx = line.indexOf('.bs"');
-            if (idx > -1) {
-                line = line.substring(0, idx) + '.brs' + line.substring(idx + 3);
-            }
-
-            //convert "text/brighterscript" to "text/brightscript"
-            line = line.replace(`"text/brighterscript"`, `"text/brightscript"`);
```

Comment:
Do you apply this transformation in your new code? I couldn't find it anywhere.

Reply:
When transpiling.